src/core/bittorrent/torrenthandle.cpp

```diff
@@ -828,16 +838,29 @@ qulonglong TorrentHandle::eta() const
     const SpeedSampleAvg speed_average = m_speedMonitor.average();
 
     if (isSeed()) {
-        if (speed_average.upload == 0) return MAX_ETA;
-
         qreal max_ratio = maxRatio();
-        if (max_ratio < 0) return MAX_ETA;
+        int max_seeding_time = maxSeedingTime();
```

Comment:
>but only half of the original code is camelCased

Rules have been developed recently. We use them in new code and changing old from time to time.